dts/bindings/sensor/aosong,dht.yaml

```diff
@@ -1,9 +1,9 @@
 # Copyright (c) 2019 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-title: Aosong DHT Digital-output Humidity and Temperature Sensor
-
 description: |
+    Aosong DHT Digital-output Humidity and Temperature Sensor
```

Comment:
Could maybe make it

    Aosong DHT (Digital-output Humidity and Temperature sensor)

to make it clear that it's explaining the acronym.

Reply:
Done

Reply:
Could make 's' lowercase since "sensor" isn't part of the acronym.